tests/unit/policyEvaluator.js

```diff
@@ -1112,6 +1112,24 @@ describe('policyEvaluator', () => {
     });
 });
 
+describe('policyEvaluator for utapi', () => {
```

Comment:
the wildcards, variables, conditions and operators are not different for utapi so no need to replicate the prior tests.